Spec/Auth.swift

```diff
@@ -2160,18 +2163,19 @@ class Auth : QuickSpec {
                     expect(params.timestamp).to(equal(NSDate(timeIntervalSince1970: 123)))
                 }
 
-                it("if not explicitly set, should be generated at the getter and stick") {
+                it("if not explicitly set, should be generated at the initializer and stick") {
                     let params = ARTTokenParams()
 
                     waitUntil(timeout: testTimeout) { done in
+                        let now = NSDate().artToIntegerMs()
+                        guard let timestamp = params.timestamp else {
```

Comment:
@mattheworiordan Yes, I changed the `TokenParams` initialiser to always include a `timestamp`.

> TokenParams object has no timestamp value unless explicitly set
I will fix it.